tuna.go

```diff
@@ -745,6 +750,21 @@ func (c *Common) CreateServerConn(force bool) error {
 				return err
 			}
 
+			entryToExitMaxPrice, exitToEntryMaxPrice, err := ParsePrice(c.ServiceInfo.MaxPrice)
+			if err != nil {
+				return err
+			}
+			if entryToExitMaxPrice > 0 || exitToEntryMaxPrice > 0 {
+				balance, err := c.Wallet.Balance()
```

Comment:
Better to use `c.Client.BalanceByAddress(c.Wallet.Address())` because it's more reliable

Reply:
Yes, modified.